src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOQuerySupport.scala

```diff
@@ -126,6 +128,7 @@ trait ElasticSearchDAOQuerySupport extends ElasticSearchDAOSupport {
       .setHighlighterFragmentSize(50)
       .setHighlighterPreTags(HL_START)
       .setHighlighterPostTags(HL_END)
+      .setHighlighterEncoder("html")
```

Comment:
Is this still necessary? If not, it might suggest one more test: ensure a search that includes "foo & bar" doesn't get escaped.